src/aeromancy/runner.py

```diff
@@ -110,13 +111,16 @@ def build_docker(
         (
             "--tag",
             docker_tag,
+            # The version tag should be updated whenever Dockerfile changes.
             "https://github.com/quant-aq/aeromancy.git#v0.1.0:docker",
         ),
     )
     docker_command = " ".join(docker_commmand_pieces)
 
     console.log(f"Running {docker_command!r}", style="info")
-    docker_status, docker_output = subprocess.getstatusoutput(docker_command)
+    docker_status, docker_output = subprocess.getstatusoutput(  # noqa: S605
```

Comment:
for your enjoyment, 118-126 is pretty much:

`docker_output = subprocess.check_output(docker_command_pieces)`

except args with spaces in them wouldn't split into two args , and the logging would be different.

No action required.

Reply:
Yeah, just kidding. It's fully subprocess.check_output now since subprocess.getstatusoutput was behaving strangely with lists-of-string-commands instead of string-commands. Also changed a few things around here to improve debugging for image building.